zerver/tests/test_markdown.py

```diff
@@ -1371,11 +1371,12 @@ def test_default_code_block_language(self) -> None:
         text = "```{}\nconsole.log('Hello World');\n```\n"
 
         # Render without default language
-        msg_with_js = markdown_convert_wrapper(text.format('js'))
+        msg_with_js = markdown_convert_wrapper(text.format('javascript'))
```

Comment:
Why did you make this change?  It looks like we were using an alias `js` for `javascript` before; I suspect that we should not have removed that test and instead changed the implementation to canonicalize aliases like this when generating `data-codehilite-class`?